libraries/AP_HAL_ChibiOS/SPIDevice.h

```diff
@@ -127,6 +124,8 @@ class SPIDevice : public AP_HAL::SPIDevice {
     static void *spi_thread(void *arg);
     static uint32_t derive_freq_flag_bus(uint8_t busid, uint32_t _frequency);
     uint32_t derive_freq_flag(uint32_t _frequency);
+    // low level transfer function
+    bool do_transfer(const uint8_t *send, uint8_t *recv, uint32_t len);
```

Comment:
`WARN_IF_UNUSED`?

The callers of this should be passing failure back up but aren't.

Reply:
done. I'm not going to add it in AP_HAL/SPIDevice.h, that would be a huge scope creep